dashboard/test/ui/features/project_gallery.feature

```diff
@@ -6,6 +6,7 @@ Background:
   And I give user "Project_Czar" project validator permission
   And I remove featured projects from the gallery
 
+@no_mobile
```

Comment:
What you have here is fine, but FYI you can also put `@no_mobile` before the Feature and then not have to put it before every Scenario